patches/net/minecraft/core/HolderLookup.java.patch

```diff
@@ -1,5 +1,14 @@
 --- a/net/minecraft/core/HolderLookup.java
 +++ b/net/minecraft/core/HolderLookup.java
+@@ -26,7 +_,7 @@
+         return this.listTags().map(HolderSet.Named::key);
+     }
+ 
+-    public interface Provider {
++    public interface Provider extends net.neoforged.neoforge.common.extensions.IHolderLookupProviderExtension{
```

Comment:
Please add a new line between the interface and the left brace.

Reply:
A new line or a space? The existing file had a space, but not a new line. Other interfaces do not add a new line for the brace like in IlevelExtension
```java
public abstract class Level extends net.neoforged.neoforge.attachment.AttachmentHolder implements LevelAccessor, AutoCloseable, net.neoforged.neoforge.common.extensions.ILevelExtension {
```

Reply:
Extra space yes, sorry.